cluster-install/operator-proxy-role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: operator-proxy-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
```

Comment:
This is only necessary until some of the RBAC YAML gets re-organized/figured out (which Nathan is working on).